projects/isy-angular-widgets/src/lib/wizard/components/wizard/wizard.component.spec.ts

```diff
@@ -278,118 +275,112 @@ describe('Integration Tests: WizardComponent with Mock Parent', () => {
     expect(wizard.height).toEqual(height);
   });
 
-  it('should have ContentChildren available', () => {
-    expect(contentChildren).not.toBeUndefined();
+  it('should have available ContentChildren', () => {
     expect(contentChildren).toBeTruthy();
   });
 
   it('should have the correct number of content children', () => {
     expect(contentChildren.length).toEqual(childrenLabels.length);
   });
 
-  it('should not have saved on init', () => {
+  it('should not be saved on init', () => {
     expectIsSaved(false);
   });
 
-  it('should not have the next step available on init', () => {
+  it('should not have an available next step on init', () => {
     expectNextStepIsAllowed(false);
   });
```

Comment:
sounds like "should never have an available step", which is wrong. 
also, it is set from outside component, rendering the test useless